pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -45,7 +45,7 @@ import (
 const (
 	// If the pod stays in unschedulableQ longer than the unschedulableQTimeInterval,
 	// the pod will be moved from unschedulableQ to activeQ.
-	unschedulableQTimeInterval = 60 * time.Second
+	DefaultUnschedulableQTimeInterval = 60 * time.Second
```

Comment:
Let's merge this const block with the one below.

Reply:
Cool! Will do!